script.js

```diff
@@ -48,26 +47,39 @@
             showHomeControl: false,
             showFullPageControl: false,
             showRotationControl: false,
-            showSequenceControl: false,
+            showSequenceControl: false
         });
 
         // https://developer.mozilla.org/en-US/docs/Web/API/Storage/getItem
         // ...returns `null` if not found. JSON.parsing `null` also returns `null`
-        let savedResponse = JSON.parse(localStorage.getItem(savedResponseKey));
+        //let savedResponse = JSON.parse(localStorage.getItem(savedResponseKey));
+        savedResponse = null;
```

Comment:
Ah, I understand that this is being done to cache-bust the saved response, but we need to keep this functionality for performance reasons. This makes it so that the new tab doesn't have to perform an API request before showing the image, which saves about 200 ms.

As an alternative, maybe there is a way to unset the saved response in `save_options`?

Reply:
Gotcha, good call. I modified the logic slightly to compare the current and saved options to see if they differ, and if so run the new response. This seems to maintain good performance!